modules/byo-vpc-multiple-az.adoc

```diff
@@ -0,0 +1,62 @@
+// Module included in the following assemblies:
+//
+// * assemblies/byo-vpc.adoc
+
+[id="byo-vpc-multiple-az_{context}"]
+= BYO VPC with multiple availability zones required fields
+
+. Create six subnets for three separate zoens as follows:
++
+|===
+| Availability zone | Subnet name | CIDR IP Address
+| A | `publicA`  | `10.0.0.0/19`
+| A | `privateA` | `10.0.96.0/19`
+| B | `publicB`  | `10.0.32.0/19`
+| B | `privateB` | `10.0.128.0/19`
+| C | `publicC`  | `10.0.64.0/19`
+| C | `privateC` | `10.0.160.0/19`
+|===
++
+[NOTE]
+====
+Each subnet is for a separate zone, each with a `public` and `private` subnet.
+====
+
+. Create a `RouteTable` that is associated with the three `public` subnets
+. Create a routing rule on the RouteTable for IP address `0.0.0.0/0` to the Internet gateway that was created.
+. Create three NAT gateways in the subnet as follows:
+.. A NAT gateway on `publicA` called `privateA` with an elastic Internet address (EIP).
+.. A NAT gateway on `publicB` called `privateB` with an EIP.
+.. A NAT gateway on `publicC` called `privateC` with an EIP.
++
+[NOTE]
+====
+AWS has a default limit of five EIPs per region.
+====
++
+. Create a `RouteTable` that is associated with the three `private` subnets as follows:
+.. Create a routing rule on the RouteTable for IP address `0.0.0.0/0` to the NAT gateway called `privateA`.
+.. Create another routing rule for IP address `0.0.0.0/0` to the NAT gateway called `privateB`.
+.. Create another routing rule for IP address `0.0.0.0/0` to the NAT gateway called `privateC`.
+
+=== OCM Configuration
+
+.Procedure
+
+. Log in to {cloud-redhat-com}.
+. Click *Create Cluster* -> *Red Hat OpenShift Dedicated* -> *Create Clusters*.
+. Select *Customer Cloud Subscription (CCS)* for the billing model.
+. Select *AWS* as the cloud infrastructure provider.
+. Enter the AWS Account ID for the `osdCcsAdmin` role on the cluster creation page.
```

Comment:
Cluster creation page should be in asterisks based on https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#api-object-formatting